### ISSUE TITLE
New cop for yoda expressions

### ISSUE THREAD

**Issue by fatkodima** [rubocop/rubocop#9222]
Inspired by https://palantir.github.io/tslint/rules/binary-expression-operand-order/

This is in addition to existing `Style/YodaCondition` cop.
```ruby
# bad
y = 1 + x

# good
y = x + 1
```

**Comment by bbatsov** [comment]
Seems like a good idea to me.

**Comment by IamRaviTejaG** [comment]
@fatkodima @bbatsov I'd like to take a jab at this.
